src/olympia/amo/templates/amo/500.html

```diff
@@ -4,15 +4,18 @@
 
 {% block content %}
 {% trans home="/" %}
-<h1>Oops!  We had an error.</h1>
+<h1>Oops! We had an error.</h1>
 
 <p>We'll get to fixing that soon.</p>
 <p>
   You can try refreshing the page, or head back to the
-  <a href="{{ home }}">Add-ons homepage</a>.
+  <a href="{{ home }}">Add-ons homepage</a>. You can also help us fix this by
+  <a href="https://github.com/mozilla/addons/issues/new">filing an issue</a> with the
+  details of how you got to this page. Make sure to include this error reference
+  number: {{ ref }}.
```

Comment:
Remove that last sentence as we don't support error references yet.

Reply:
I've made the change.